android/src/com/mapswithme/maps/editor/data/TimeFormatUtils.java

```diff
@@ -39,9 +39,17 @@ public static String formatShortWeekday(@IntRange(from = 1, to = 7) int day)
     return sShortWeekdays[day];
   }
 
-  public static String formatWeekdays(@NonNull Timetable timetable)
+  public static String formatWeekdaysRange(int startWeekDay, int endWeekDay)
   {
     refreshWithCurrentLocale();
+    if (startWeekDay == endWeekDay)
```

Comment:
Chronological week schedule has little bit different format comparing to `Timetable` class. It doesn't have array of days like `Timetable.weekdays`. Each row in week schedule is either single day schedule or days range schedule.

* Su - 10:00-15:00 (single day)
* Mo-Fr - 09:00-18:00 (days range)
* Sa - 10:00-15:00 (single day)

So I store start and end day for each row. But in case of a single day we have `startWeekDay == endWeekDay`.